src/resampling.jl

```diff
@@ -219,9 +240,9 @@ function resample_czt(arr::AbstractArray{T,N}, rel_zoom; shear=nothing, shear_di
                 FourierTools.apply_shift_strength!(f_res, f_res, shifts, d, sd, -myshear, fix_nyquist)
             end
             if T<:Real
-                f_res = real(FourierTools.czt_1d(f_res, my_zoom, d))
+                f_res = real(FourierTools.czt_1d(f_res, my_zoom, d; remove_wrap=remove_wrap, pad_value=pad_value))
             else
-                f_res = T.(FourierTools.czt_1d(f_res, my_zoom, d))
+                f_res = T.(FourierTools.czt_1d(f_res, my_zoom, d; remove_wrap=remove_wrap, pad_value=pad_value))
```

Comment:
Why is the `T.` needed here?
Isn't the result complex anyways?

Reply:
agreed. removed it.